dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/registry/ZookeeperNodeManager.java

```diff
@@ -38,33 +35,37 @@
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 
-import static org.apache.dolphinscheduler.common.Constants.DEFAULT_WORKER_GROUP;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 /**
- *  zookeeper node manager
+ * zookeeper node manager
```

Comment:
Is this file changes related with this pr?
If not related, please revert.

Reply:
> Is this file changes related with this pr?
> If not related, please revert.
I've optimized this part
```
String host = org.apache.commons.lang.StringUtils.substringAfterLast(path, "/");
```
because code style,Become less obvious ,lint 226 ,265

Reply:
Extract the exact host information in the path of zookeeper

Reply:
modified